oauth_mgmt/models_test.py

```diff
@@ -0,0 +1,25 @@
+"""
+Model tests.
+"""
+from datetime import timedelta
+from django.utils.timezone import now
+import pytest
+from .models import BackingInstance
+
+
+def test_backing_instance_str():
+    "__str__ returns url"
+    assert str(BackingInstance(instance_url='https://edx.org/')) == "https://edx.org/"
+
+
+@pytest.mark.parametrize("incoming,expected", [
+    (now(), True),
+    (now() - timedelta(days=1), True),
+    (now() + timedelta(hours=1), True),  # no padding
```

Comment:
I have it setup to require 2 hours of padding on the expiration. There needs to be some padding so we don't have it expiring between checking that the token is valid and trying to use it.